noq_django/rest_api/api/user_api.py

```diff
@@ -59,29 +60,35 @@ def list_available(request, selected_date: str):
     
     return [{"host": host, "products": products} for host, products in hostproduct_dict.items()]
 
-@router.post("/request_booking", response=BookingSchema, tags=["user-booking"])
-def request_booking(request, booking_data: BookingPostSchema):
-
+@router.post("/booking", response=BookingSchema, tags=["user-booking"])
```

Comment:
We should not change the API that is in use. We should leave the old API and create a new one with the new name and remove the old one after frontend has been updated to use the new API.

Reply:
I agree with you. I didn't think we used it yet. But if it is in use we should definitely do it like you said